src/libraries/System.IO.FileSystem/tests/File/AppendAllBytes.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO.Tests;
+using System.Linq;
+using System.Text;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public class File_AppendAllBytes : FileSystemTest
+    {
+
+        [Fact]
+        public void NullParameters()
+        {
+            string path = GetTestFilePath();
+
+            Assert.Throws<ArgumentNullException>(() => File.AppendAllBytes(null, new byte[0]));
+            Assert.Throws<ArgumentNullException>(() => File.AppendAllBytes(path, null));
+        }
+
+        [Fact]
+        public void InvalidParameters()
+        {
+            Assert.Throws<ArgumentException>(() => File.AppendAllBytes(string.Empty, new byte[0]));
+        }
+
+
+        [Fact]
+        public void AppendAllBytes_WithValidInput_AppendsBytes()
+        {
+            string path = GetTestFilePath();
+
+            byte[] initialBytes = Encoding.UTF8.GetBytes("bytes");
+            byte[] additionalBytes = Encoding.UTF8.GetBytes("additional bytes");
+
+            File.WriteAllBytes(path, initialBytes);
+            File.AppendAllBytes(path, additionalBytes);
+
+            byte[] result = File.ReadAllBytes(path);
+
+            byte[] expectedBytes = initialBytes.Concat(additionalBytes).ToArray();
+
+            Assert.True(result.SequenceEqual(expectedBytes));
+        }
+
+
+        [Fact]
+        public void EmptyContentCreatesFile()
+        {
+            string path = GetTestFilePath();
+            File.AppendAllBytes(path, new byte[0]);
+            Assert.True(File.Exists(path));
+            Assert.Empty(File.ReadAllBytes(path));
+        }
+
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsFileLockingEnabled))]
+        public void OpenFile_ThrowsIOException()
+        {
+            string path = GetTestFilePath();
+            byte[] bytes = Encoding.UTF8.GetBytes("bytes");
+
+            using (File.Create(path))
+            {
+                Assert.Throws<IOException>(() => File.AppendAllBytes(path, bytes));
+                Assert.Throws<IOException>(() => File.ReadAllBytes(path));
```

Comment:
This is already tested in https://github.com/dotnet/runtime/blob/00f1e0c94a3853f65d153179c1981d5b3f01c5e6/src/libraries/System.IO.FileSystem/tests/File/ReadWriteAllBytes.cs#L79
```suggestion
```